docsite/source/do-notation.html.md

```diff
@@ -74,7 +74,20 @@ class CreateAccount
 end
 ```
 
-Both snippets do the same thing yet the second one is a lot easier to deal with. All what `Do` does here is prepending `CreateAccount` with a module which passes a block to `CreateAccount#call`. That simple.
+Both snippets do the same thing yet the second one is a lot easier to deal with. All what `Do` does here is prepending `CreateAccount` with a module which passes a block to `CreateAccount#call`.
+
+#### yield
+A little more on `yield`. It will return a `Result` (remember that's either a Success or Failure object) and **if** the `Result` is a Success object, then yield will unpack it.
```

Comment:
"It will accept a `Result`...", everything else is 💯 accurate 🎉

Reply:
Thanks for that! I've updated it now :)